src/Console/Command/DescribeCommand.php

```diff
@@ -0,0 +1,227 @@
+                $fixer->configure($codeSample[1]);
+                $fixer->fix(new StdinFileInfo(), $tokens);
+                $new = $tokens->generateCode();
+                $diff = $differ->diff($old, $new);
+
+                if (null === $codeSample[1]) {
+                    $output->writeln(sprintf(' * Example #%d.', $index + 1));
+                } else {
+                    $output->writeln(sprintf(' * Example #%d. Fixing with configuration: <comment>%s</comment>.', $index + 1, $this->arrayToText($codeSample[1])));
+                }
+                $output->writeln($this->prepareDiff(true, $diff));
+                $output->writeln('');
+            }
+        }
+
+        if (!$description) {
+            $output->writeln(sprintf('<question>This rule is not yet described, do you want to help us and describe it?</question>'));
+            $output->writeln('Contribute at <comment>https://github.com/FriendsOfPHP/PHP-CS-Fixer</comment>');
+            $output->writeln('');
+        }
+    }
+
+    private function describeSet(InputInterface $input, OutputInterface $output, $name)
+    {
+        $ruleSet = new RuleSet(array($name => true));
+        $rules = $ruleSet->getRules();
+        ksort($rules);
+
+        $fixerFactory = new FixerFactory();
+        $fixers = array();
+
+        foreach ($fixerFactory->registerBuiltInFixers()->getFixers() as $fixer) {
+            $fixers[$fixer->getName()] = $fixer;
+        }
+
+        $output->writeln(sprintf('<info>Description of %s set.</info>', $name));
+        $output->writeln('');
+
+        $help = '';
+
+        $count = count($rules) - 1;
+        foreach ($rules as $rule => $config) {
+            $help .= sprintf(
+                " * <info>%s</info>%s\n   | %s\n%s\n",
+                $rule,
+                $fixers[$rule]->isRisky() ? ' <error>[risky]</error>' : '',
+                $fixers[$rule]->getDescription(),
+                true !== $config ? sprintf("   <comment>| Configuration: %s</comment>\n", $this->arrayToText($config)) : ''
+            );
+        }
+
+        $output->write($help);
+    }
+
+    /**
+     * @param FixerInterface $fixer
+     *
+     * @return bool
+     */
+    private function isFixerConfigurable(FixerInterface $fixer)
+    {
+        try {
+            $fixer->configure(array());
+
+            return true;
+        } catch (UnallowedFixerConfigurationException $e) {
+            return false;
+        } catch (\Exception $e) {
+            return true;
+        }
+    }
+
+    private function arrayToText(array $data)
+    {
+        static $replaces = array(
+            '#\r|\n#' => '',
+            '#\s{1,}#' => ' ',
+            '#array\s*\((.*)\)#s' => '[$1]',
+            '#\[\s+#' => '[',
+            '#,\s*\]#' => ']',
+            '#\d+\s*=>\s*#' => '',
+        );
+
+        return preg_replace(
+            array_keys($replaces),
+            array_values($replaces),
+            var_export($data, true)
```

Comment:
No, i dont want big `VarDumper` package, which will produce somehow pretty and colourful representation.
Here we only want to have simple, small, one-lined config representation.